### PR TITLE
Use ubuntu as base image for docker containers (8.x)

### PR DESCRIPTION
**Pull request by nagyo**
- simplified builder stage
- more secure file ownership settings
- use tini to init

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -23,48 +21,58 @@ ARG GIT_REVISION
 ARG GIT_BRANCH
 ARG PRODUCT_NAME="Snow Owl OSS"
 ARG REPOSITORY_URL="https://github.com/b2ihealthcare/snow-owl"
+ARG CI_PASSED=""
+ARG WIN_CLIENT_URL=""
+ARG MAC_CLIENT_URL=""
```
Comment (cmark) [inline docker/Dockerfile]: Do we really need the client urls here?

**Review by cmark — CHANGES_REQUESTED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (nagyo) [reply docker/Dockerfile]: Nope, good catch :thumbsup:

**Review by cmark — APPROVED** [review]
🥓

**Review by apeteri — APPROVED** [review]
👍

**Pull request merged**